### PR TITLE
featuer donators team chat

### PR DESCRIPTION
**Pull request by dgorkavyi**
другийСкрін

### CODE REVIEW

**Comment by dgorkavyi** [comment]
It is just a component without socket functionality yet

File: frontend/src/components/Chat/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const littleDateFormate = (date: Date) => {
+  const locale = 'en-US';
+  const timeZone = { timeZone: 'UTC' };
+  const currDateStr = new Date().toLocaleString(locale, timeZone).split(',');
```
Comment (kubovych) [inline frontend/src/components/Chat/helpers.ts]: Why we use USA time? May let the browser pick up the local time automatically

File: frontend/src/components/Chat/helpers.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  const argDateStr = date.toLocaleString(locale, timeZone).split(',');
+  return currDateStr[0] === argDateStr[0]
```
Comment (kubovych) [inline frontend/src/components/Chat/helpers.ts]: need to cut a seconds, they overload the interface

**Review by kubovych — CHANGES_REQUESTED** [review]

File: frontend/src/components/Chat/ChatContent.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { littleDateFormate } from './helpers';
+import { Message } from './MessageInterface';
+import classes from './styles.module.scss';
+
+const messageMapper = (mess: Message) => (
```
Comment (turfle) [inline frontend/src/components/Chat/ChatContent.tsx]: what is mess? i don't think that is a good variable name.

Screenshot from 2021-09-03 10-28-26

File: frontend/src/components/Chat/helpers.ts (hunk 1 above)
Comment (yevhenii-moroziuk) [reply frontend/src/components/Chat/helpers.ts]: use our time helper

**Comment by dgorkavyi** [comment]
labels looks like: 
image

**Review by yevhenii-moroziuk — APPROVED** [review]

**Pull request merged**